### PR TITLE
add online version

### PR DESCRIPTION
**Pull request by mj-e-boc**
- Deploy portfolio
- Update readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +52,7 @@ Built with HTML and CSS
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-Link to be added
-
-- [Live Demo Link]()
+- [Live Demo Link](https://mj-e-boc.github.io/)
```
Comment (Johnadibe) [inline README.md:55]: > - [ ] Upon reviewing your hosted website, I observed that the contact form on the desktop screen is not functioning as intended. I kindly request you to watch the provided video screencast, which demonstrates the issue, and proceed with rectifying it. Having a working contact form enables visitors to easily reach out to you, allowing for effective communication and potential collaboration or inquiries and it also improves user experience.☺Moreover, it is recommended to set the maximum length for the textarea to 500, while not enforcing a minimum length requirement.👍
> 
> 
> https://github.com/mj-e-boc/M-J-Portfolio/assets/40009354/4d046bfb-3b4a-44f0-b1b8-fce91b892e0d
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @mj-e-boc👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You deployed your website using GitHub pages👍🏽


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @mj-e-boc , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

🚫 Oops! It seems that there was an issue with your project submission. Upon reviewing your project, I noticed that this branch is intended for deploying your code using GitHub Pages. However, you requested a check for the "**[Portfolio: accessibility](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)**" milestone. It is mandatory to create a new branch for each milestone, as per the guidelines provided in the [Github flows](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).

To ensure proper organization and adherence to the Git workflow, please create a new branch specifically for the "Portfolio: accessibility" milestone. This will help maintain a clear and structured version control history for your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Alusp — APPROVED** [review]
Hi @mj-e-boc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Highlights

- [ ] Use Live Demo Correctly :green_circle: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**